frontend/src/components/common/Label.tsx

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
 import './Label.scss';
 interface ILabelProps {
-  children?: string;
+  content?: string;
```

Comment:
using children as a string is confusing given the react naming convention for nested child components.